build/release/containers.sh

```diff
@@ -52,16 +52,60 @@ build() {
 
     [[ ${os} == "linux" ]] || return 0
 
-    tmpdir=$(mktemp -d)
-    trap "rm -fr $tmpdir" EXIT
-    cat <<EOF > $tmpdir/Dockerfile
+    echo "Building the rookd container"
+    tmpdir1=$(mktemp -d)
+    trap "rm -fr $tmpdir1" EXIT
+    cat <<EOF > $tmpdir1/Dockerfile
 FROM alpine:3.5
 RUN apk add --no-cache gptfdisk util-linux coreutils e2fsprogs
 COPY root /
 ENTRYPOINT ["/usr/bin/rookd"]
 EOF
-    build_artifact $os $arch $tmpdir rook/rookd rookd
-    rm -fr $tmpdir
+    build_artifact $os $arch $tmpdir1 rook/rookd rookd
+    rm -fr $tmpdir1
+
+    echo "Building the rook-client container"
+    tmpdir2=$(mktemp -d)
+    trap "rm -fr $tmpdir2" EXIT
+    cat <<EOF > $tmpdir2/Dockerfile
+FROM ubuntu
```

Comment:
the change to base the client image on alpine is turning into a big change to remove a dependency on bash. I think it best to split out that change into a separate PR and keep the container on ubuntu for this PR.